sdk/src/sanitize.rs

```diff
@@ -0,0 +1,18 @@
+pub enum SanitizeError {
```

Comment:
use thiserror?

Reply:
@jackcmay what's that?

Reply:
https://github.com/solana-labs/solana/blob/c5a7df9221fa93c4f0d085192815dde6cd5ed2d0/sdk/src/instruction.rs#L13

https://docs.rs/thiserror/1.0.15/thiserror/